sb3_contrib/common/distributions.py

```diff
@@ -0,0 +1,146 @@
+from typing import Any, Dict, Optional, Tuple
+
+import gym
+import numpy as np
+import torch as th
+from gym import spaces
+from stable_baselines3.common.distributions import (
+    BernoulliDistribution,
+    DiagGaussianDistribution,
+    Distribution,
+    MultiCategoricalDistribution,
+    StateDependentNoiseDistribution,
+)
+from stable_baselines3.common.preprocessing import get_action_dim
+from torch import nn
+from torch.distributions import Categorical
+
+
+class MaskableCategorical(Categorical):
```

Comment:
can't you merge this one with `CategoricalDistribution` ?

Reply:
How so? Do you mean combining into a single class, or just moving this to be an inner class?

Reply:
It was indeed reusable, so resolving for now.